### PR TITLE
XML elimination blogpost

### PR DESCRIPTION
**Pull request by ostackbrian**
bio:
 Brian Rosmaita is a software developer on the Rackspace team that works
 on Cloud Servers and Cloud Images (that's Nova and Glance in OpenStack
 language).  Upstream at OpenStack, he's a driver for the Glance and
 Searchlight teams.  You can find him on Freenode as rosmaita.  Use @br14nr
 if you want to tweet at him.

social:
 - Freenode: rosmaita
 - Twitter: @br14nr

I accept redaction, editing or structure changes by technical writer staff

desired publication date: June 9, 2015

### CODE REVIEW

File: src/site_source/_posts/2015-06-09-goodbye-xml.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+the response format you can expect can be found in the [Compute API
+documentation](http://docs.rackspace.com/servers/api/v2/cs-devguide/content/ch_preface.html).
+Make sure that your request either contains no "Accept" header, or that
+it explicitly specifies "Accept: application/json" in the request
+header.
+
+How easy a transition this will be depends upon what scripting language
+you are using. Some languages (Python, for example) handle JSON
+serialization/deserialization much more easily than others. If you need
+to do a major rewrite or your scripts, we strongly recommend considering
```
Comment (catlook) [inline src/site_source/_posts/2015-06-09-goodbye-xml.md]: ... major rewrite of your scripts ...

File: src/site_source/_posts/2015-06-09-goodbye-xml.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+version of an SDK that still used XML.  The most recent versions of
+currently maintained OpenStack SDKs don't use XML, so an upgrade will
+fix this problem.
+
+## Goodbye XML!
+
+In conclusion, mark July 20, 2015 on your calendar as the last day of
+XML support for the Rackspace Cloud Servers API.  If you have questions
+or comments about this transition, feel free to post a comment in the
+[Compute Feedback Forum](https://feedback.rackspace.com/forums/298152-compute).
```
Comment (catlook) [inline src/site_source/_posts/2015-06-09-goodbye-xml.md]: [Edit checking done :)]

**Comment by rgbkrk** [comment]
I'm all for the death of XML.

File: src/site_source/_posts/2015-06-09-goodbye-xml.md (hunk 1 above)
Comment (ostackbrian) [inline src/site_source/_posts/2015-06-09-goodbye-xml.md]: thanks for catching this, will correct right away.

**Comment by ostackbrian** [comment]
@rgbkrk I'm happy to do my part!

**Comment by rgbkrk** [comment]
Care to synchronize :shipit: @smashwilson ?

**Pull request merged**